src/lib/rule-finder.js

```diff
@@ -33,16 +33,41 @@ function _getCurrentRules(config) {
   return Object.keys(config.rules)
 }
 
+function _pluginNameToModule(name) {
+  var scopedRegex = /(@[^/]+)\/(.+)/
+  var match = scopedRegex.exec(name)
+
+  if (match) {
+    return match[1] + '/' + 'eslint-plugin-' + match[2]
+  }
+
+  return 'eslint-plugin-' + name
+}
+
+function _pluginNameToRulePrefix(name) {
+  var scopedRegex = /(@[^/]+)\/(.+)/
+  var match = scopedRegex.exec(name)
+
+  if (match) {
+    return match[2]
+  }
+
+  return name
+}
+
```

Comment:
`_pluginNameToModule` and `_pluginNameToRulePrefix` is almost same other than the return value, could you combine them in to one? 

Something like so ...
```js
function _normalizePluginName(name) {
  var scopedRegex = /(@[^/]+)\/(.+)/
  var match = scopedRegex.exec(name)

  if (match) {
    return {
      module: match[1] + '/' + 'eslint-plugin-' + match[2],
      prefix: match[2]
    }
  }

  return {
    module: 'eslint-plugin-' + name
    prefix: name
  }
}
```